forge/routes/api/search.js

```diff
@@ -149,4 +151,96 @@ module.exports = async function (app) {
             results: []
         })
     })
+
+    /**
+     * Search API
+     *
+     * GET /api/v1/search/instances
```

Comment:
Is there a reason for this to be a new search endpoint rather than add something like `scope=instances` to the existing `/search` endpoint?

Reply:
The main reason was to keep the query as lightweight as possible and avoid pulling in unnecessary relations. 

It was also much easier to trim the payload with a dedicated endpoint than to adapt the existing one and limit the `byTeam` finders without side effects.

Reply:
I was thinking the underluing implmentation would be what you've done already, just a big `if scope=instances` then do one way, otherwise do the existing.

But I'll not block on that.